docs/developers/developing_iree/testing_guide.md

```diff
@@ -219,13 +219,13 @@ drivers).
 With CMake, run this from the build directory:
 
 ```shell
-$ ctest -R tests/e2e/xla_ops/check_vmvx_vmvx_floor.mlir
+$ ctest -R tests/e2e/xla_ops/check_vmvx_local-task_floor.mlir
```

Comment:
it's annoying to have mixed -/_ in these - can we just make them all _?

Reply:
`check_llvm-cpu_local-task` follows `check_${COMPILER_TARGET}_${HAL_DEVICE}`, with `llvm-cpu` and `local-task` names matching what the compiler and runtime expect exactly. We could diverge from the code for consistency in build system target naming, yeah.

Reply:
I'd rather we have consistency with the driver naming, personally. Then a search for "llvm-cpu" turns up all usages and we don't have to remember it's sometimes "llvm_cpu". Additionally, this gives us delimiters such that "local-task" is clearly all one thing and it's not "vmvx_local".

Reply:
@benvanik how do you feel about that? I like being able to find-replace too, even though target name style is important too...